### PR TITLE
Sign Powershell scripts for VS

### PR DESCRIPTION
**Pull request by dan-shaw**

### CODE REVIEW

File: scripts/addPoshVcpkgToPowershellProfile.ps1 [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[CmdletBinding()]
```
[the remark this answers is not in the document]
Comment (BillyONeal) [reply scripts/addPoshVcpkgToPowershellProfile.ps1:1]: IMO anything that we are signing should have the authoritiative copy be here and updating the copies in Microsoft/vcpkg if necessary be part of the release process. That way the things we're signing actually are in the internal repo.

File: azure-pipelines/signing.yml [hunk 2]
```diff
@@ -341,6 +358,10 @@ jobs:
           move "$(Build.ArtifactStagingDirectory)\staging\vcpkg-muslc" "$(Build.ArtifactStagingDirectory)\drop\vcpkg-muslc"
           move "$(Build.ArtifactStagingDirectory)\staging\vcpkg-standalone-bundle.tar.gz" "$(Build.ArtifactStagingDirectory)\drop\vcpkg-standalone-bundle.tar.gz"
 
+          move /Y "$(Build.ArtifactStagingDirectory)\staging\applocal.ps1" "$(Build.ArtifactStagingDirectory)\vs-insertion\staging\scripts\buildsystems\msbuild\applocal.ps1"
```
Comment (BillyONeal) [inline azure-pipelines/signing.yml:361]: I think you need to edit mint-standalone-bundle.ps1 to pick up these signed copies so they also end up in vcpkg-standalone-bundle.tar.gz. Other than that this looks great to me!

**Review by BillyONeal — CHANGES_REQUESTED** [review]

File: azure-pipelines/signing.yml [hunk 3]
```diff
@@ -17,13 +17,21 @@ parameters:
   displayName: vcpkg Base Version (default is today's date in ISO 8601)
   type: string
   default:
-
+- name: PublishLocation
```
Comment (strega-nil-ms) [inline azure-pipelines/signing.yml:20]: Could this be a different name? Something like `PublishedTo` or smth. `PublishLocation` makes me think of a URI to publish to or something.

**Comment by BillyONeal** [comment]
I am working on resolving the code review feedback thus far.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
@dan-shaw I pushed fixes for the things I indicated, including ensuring the signed scripts are also in the "standalone bundle" tarball. Please check that you're happy. (I also fixed @strega-nil 's PR comment about the name of the radio button to do partial publishes)

**Pull request merged**